imagepipeline-base/src/main/java/com/facebook/imageformat/ImageFormat.java

```diff
@@ -32,7 +35,8 @@
      * @return the image format or null if unknown
      */
     @Nullable
-    ImageFormat determineFormat(byte[] headerBytes, int headerSize);
+    ImageFormat determineFormat(byte[] headerBytes, int headerSize, InputStream is)
```

Comment:
I can read the first N bytes, but I'm not guaranteed to find the header tag of the SVG file if it starts, for example, at the 110th byte and the "max header length" is less than 100 bytes. That said, and after further review of the SVG formatting standards, it's probably best to either (1) leave the original code as it is or (2) detect some of the tags that commonly precede an SVG header tag ("<?xml" and "<!DOCTYPE") and write in a more descriptive Exception message for those scenarios. In both cases, we would ultimately expect the user to provide an image that starts with the "<svg" tag.

Thoughts?